articles/spring-cloud/how-to-enterprise-build-service.md

```diff
@@ -69,11 +69,11 @@ Select **Add** to create a new builder. The image below shows the resources you
 
 :::image type="content" source="media/enterprise/how-to-enterprise-build-service/builder-create.png" alt-text="Screenshot of 'Add Builder' pane." lightbox="media/enterprise/how-to-enterprise-build-service/builder-create.png":::
 
-You can also edit a custom builder. You can update the buildpacks or the [OS Stack](https://docs.pivotal.io/tanzu-buildpacks/stacks.html), but the builder name is read only.
+You can also edit a custom builder when the builder is not used in the a deployment. You can update the buildpacks or the [OS Stack](https://docs.pivotal.io/tanzu-buildpacks/stacks.html), but the builder name is read only.
```

Comment:
Add two sections:
1. Use a builder to deploy apps 
if not specifying builder, then use default builder;
if specifying a builder name, then use that builder.
3. Maintain a builder
to edit a builder, xxx
to delete a builder, xxx
Tips for you to update a builder for your apps to consume:
1. add a new builder with the new configs
2. redeploy your apps with the new builder.

Reply:
About Manage service binding section, add a notice on the top to tell the prerequisite to do so:
1. in a new builder
2. in an existing builder without any app deployment linking to it.